modules/tinymce/src/core/test/ts/browser/paste/InternalClipboardTest.ts

```diff
@@ -9,10 +10,20 @@ import { EditorEvent } from 'tinymce/core/api/util/EventDispatcher';
 import * as InternalHtml from 'tinymce/core/paste/InternalHtml';
 import TablePlugin from 'tinymce/plugins/table/Plugin';
 
+interface ProcessEventExpectedData {
+  internal: boolean;
+  content: string;
+}
+
+interface InputEventExpectedData {
+  data: string;
+}
+
 describe('browser.tinymce.core.paste.InternalClipboardTest', () => {
   let dataTransfer: DataTransfer | undefined;
   let lastPreProcessEvent: EditorEvent<PastePreProcessEvent> | undefined;
   let lastPostProcessEvent: EditorEvent<PastePostProcessEvent> | undefined;
+  let lastInputEvent: EditorEvent<InputEvent> | undefined;
```

Comment:
These undefined things might predate katamari. I think it better to use singletons.